optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -2300,33 +2537,26 @@ def preprocess_inputs(
         processor: Optional[AutoImageProcessor] = None,
         tokenizer: Optional[PreTrainedTokenizer] = None,
         config: Optional[PretrainedConfig] = None,
+        video: Optional["VideoInput"] = None,
     ):
         if processor is None:
             raise ValueError("Processor is required.")
+        conversation = [
+            {
+                "role": "user",
+                "content": [
+                    {"type": "text", "text": text},
+                ],
+            }
+        ]
         if image is not None:
-            conversation = [
-                {
-                    "role": "user",
-                    "content": [
-                        {
-                            "type": "image",
-                        },
-                        {"type": "text", "text": text},
-                    ],
-                }
-            ]
-        else:
-            conversation = [
-                {
-                    "role": "user",
-                    "content": [
-                        {"type": "text", "text": text},
-                    ],
-                }
-            ]
+            conversation[0]["content"].insert(0, {"type": "image"})
+        if video is not None:
+            conversation[0]["content"].insert(0, {"type": "video"})
```

Comment:
Shouldn't `_OVQwen2VLForCausalLM` and `_OVQwen2_5_VLForCausalLM` now have `export_feature="video-text-to-text"`?

Reply:
there is no need in it. I added video-text-to-text only for auto task detection using pipeline tag for hf hub. for llava-next-video this field contains video-text-to-text, for qwen2vl and qwen2.5vl - image-text-to-text